### DISCUSSION TITLE
เพิ่ม Contributors ในหน้า Landing page

### DISCUSSION THREAD

**Discussion by mrthiti** [Naga-DAO/naga-mint-workspace#9] [Announcements]
สวัสดีครับทุคน ตอนนี้ Naga DAO จะเพิ่ม Section ที่เป็นข้อมูลเกี่ยวกับ Contributors ในหน้า Landing Page ตรงนี้ครับ

<img width="500" alt="Screen Shot 2565-04-14 at 11 12 44 AM" src="https://user-images.githubusercontent.com/7018802/163313426-972f6f86-f21c-47aa-b135-9d02f1360869.png">

ดังนั้นก่อนที่เราจะมา Dev อยากจะรบกวนทุกคนหา Design เพื่อมาเป็นตัวอย่างในการ Dev ครับ แล้วเดี๋ยวเรามาโหวด แล้วเริ่ม Dev กันครับ

สามารถ Comment มาได้เลยครับ

**Comment by mrthiti** [comment]
ผมเสนออันนี้ครับ https://svelte.dev/

<img width="1306" alt="image" src="https://user-images.githubusercontent.com/7018802/163317976-598d94e7-ca4f-4fe5-b84c-c332c4ad3162.png">

**Reply by mojisejr** [reply]
แบบนี้ก็ดีเหมือนกันนะครับ เวลา contributors เยอะๆ จะได้ไม่ดูรกครับผม

**Comment by PattoMotto** [comment]
1) https://www.lateral-inc.com/about.html ผมชอบอันนี้ครับ
![lateral-team-page](https://user-images.githubusercontent.com/1745000/163684391-e635b4c4-1def-4584-a9e7-2485c6fb2f49.gif)


2) https://www.etsy.com/team หรือถ้าคนเยอะมากคงต้องแบบนี้ 🙈
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/1745000/163684143-64192a1b-615f-4797-8055-5475abeae7cf.png">

3) Kickstarter (version เก่า)
![kickstarter-meet-the-team](https://user-images.githubusercontent.com/1745000/163684370-425060dc-6a9f-4f1c-8dc8-7882a2b947cd.gif)

**Reply by mrthiti** [reply]
หมายเลข 1 เท่ห์ดีนะ แต่อาจจะต้องถ่ายรูปกันหลายชอทหน่อย 555

**Comment by mrthiti** [comment]
มาโหวดกันครับทุกคน --> #12
